bundle/01_bundle_intro.ipynb

```diff
@@ -153,10 +153,10 @@
       "    \"changelog\": {\n",
       "        \"0.0.1\": \"Initial version\"\n",
       "    },\n",
-      "    \"monai_version\": \"1.2.0\",\n",
-      "    \"pytorch_version\": \"2.0.0\",\n",
-      "    \"numpy_version\": \"1.23.5\",\n",
-      "    \"optional_packages_version\": {},\n",
+      "    \"monai_version\": \"1.4.0rc8+1.gab3ee18ba\",\n",
```

Comment:
Maybe this should just be 1.4.0 if there isn't a need for that specific commit version of the code.

Reply:
Yes, you are right, I will change this one.